test/functionalTest/cases/2015_notification_templates/notification_templates_payload_as_TEXT.test.DISABLED

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
We don't typically use UPPERCASE in .test name. Thus, better "text" than "TEXT". Same for JSON / json.